code/game/objects/structures/grille.dm

```diff
@@ -289,9 +282,6 @@
 		color = "#960000"
 		animate(src, color = previouscolor, time = 8)
 
-/obj/structure/grille/ratvar/ratvar_act()
```

Comment:
This naming itself just shows how bad it is, the syntax linter cant even handle it

Reply:
What